backend/LexBoxApi/Services/PermissionService.cs

```diff
@@ -100,6 +100,8 @@ public async ValueTask<bool> CanViewProjectMembers(Guid projectId)
         if (User is not null && User.Role == UserRole.admin) return true;
         // Project managers can view members of their own projects, even confidential ones
         if (await CanManageProject(projectId)) return true;
+        // non members can't view project members
+        if (User?.IsProjectMember(projectId) != true) return false;
```

Comment:
I think I'd rather not, just because that's a much larger pool of users. Also this would be showing users who have been managers for a long time (I don't want it to be a consideration for making their project non confidential), since orgs are new and there will be less of them I feel better about making them public.